tachyon/zk/plonk/constraint_system/constraint_system.h

```diff
@@ -604,6 +783,32 @@ class ConstraintSystem {
     return *max_required_degree;
   }
 
+  // Degree of lookup without inputs
+  constexpr static size_t ComputeBaseDegree(size_t table_degree) {
+    // φᵢ(X) = fᵢ(X) + α
+    // τ(X) = t(X) + α
+    //
+    // LHS = τ(X) * Π(φᵢ(X)) * (ϕ(gX) - ϕ(X))
+    // ↪ degree(LHS) = table_degree + inputs_degree(= 0) + 1
+    //               = table_degree + 1
+    //
+    // RHS = τ(X) * Π(φᵢ(X)) * (∑ 1/(φᵢ(X)) - m(X) / τ(X))
+    // ↪ degree(RHS) = table_degree
+    //
+    // (1 - (l_last(X) + l_blind(X))) * (LHS - RHS)
+    // ↪ degree = lhs_degree + 1 = table_degree + 2
```

Comment:
I'm not too sure if table_degree should be |table_degree|. What do you think?
```suggestion
    // ↪ DEG(LHS) = table_degree + inputs_degree(= 0) + 1
    //               = table_degree + 1
    //
    // RHS = τ(X) * Π(φᵢ(X)) * (∑ 1/(φᵢ(X)) - m(X) / τ(X))
    // ↪ DEG(RHS) = table_degree
    //
    // (1 - (l_last(X) + l_blind(X))) * (LHS - RHS)
    // ↪ degree = DEG(LHS) + 1 = table_degree + 2